### ISSUE TITLE
Task 'getdevicedata' does not return anything

### ISSUE THREAD

**Issue by ehaligow** [opencomputeproject/HWMgmt-DeviceMgr-DeviceManager#20]
After executing 'getdevicedata' task, nothing happens.
```
intel@ubuntu:functional_test$ ./dm getdevicedata 192.168.40.247:8888:admin:GET:/redfish/v1/AccountService/Accounts/3

intel@ubuntu:functional_test$ ./dm getdevicedata 192.168.40.247:8888:admin:GET:/redfish/v1/AccountService/Accounts

intel@ubuntu:functional_test$ ./dm getdevicedata 192.168.40.247:8888:admin:GET:/redfish/v1/Chassis
```

Here is the log:
```
time="25-04-2022 09:22:14.888423" level=info msg="Received GetDeviceData"
time="25-04-2022 09:22:25.125305" level=info msg="Received GetDeviceData"
time="25-04-2022 09:23:11.730752" level=info msg="Received GetDeviceData"
time="25-04-2022 09:23:27.981166" level=info msg="Received GetDeviceData"
time="25-04-2022 09:23:41.986420" level=info msg="Received GetDeviceData"
time="25-04-2022 09:26:30.399268" level=info msg="Received GetDeviceData"
time="25-04-2022 09:26:34.960931" level=info msg="Received GetDeviceData"
time="25-04-2022 09:26:47.006144" level=info msg="Received GetDeviceData"

```

**Comment by JasonscHuang** [comment]
Do you enable the retrieving data from the device periodically? If you enabled it, the device data will place in the DM. You are able to get the device data using that command without sending query requests to the device in real-time.

**Comment by ehaligow** [comment]
I enabled the retrieving data from the device periodically, but the result was the same. However, I discovered what was the issue - I tried to run task getdevicedata, providing wrong argument. The last argument should be only URI, while I also provided http method (in this case GET method). So, the issue is not an issue anymore ;). 

However, I didn't get any error, neither in logs nor in terminal. Shouldn't DM log something in cases like this?
